Sources/GDBRemote/DebugSessionImpl.cpp

```diff
@@ -572,6 +572,29 @@ ErrorCode DebugSessionImpl::onXferRead(Session &session,
   return kSuccess;
 }
 
+ErrorCode DebugSessionImpl::onSetStdFile(Session &, int fileno,
+                                         std::string const &path) {
+  bool success = false;
+  switch (fileno) {
+  case 0:
+    success = _spawner.redirectInputToFile(path);
+    break;
+  case 1:
+    success = _spawner.redirectOutputToFile(path);
+    break;
+  case 2:
```

Comment:
For some reason, the ProcessSpawner methods seem to use hardcoded 0,1,2. But using constants seems reasonable.